publish/build.gradle

```diff
@@ -19,7 +19,7 @@
  */
 
 ext {
-  version = "4.0.0"
+  version = "5.0.0-beta1"
```

Comment:
Hmm, beta because we haven't tested it rigorously yet?

Reply:
I meant to mark that as an open question. I would like to play with it a little bit locally before having a release because this is a pretty substantial change in some ways but I am happy to keep it as 5.0.0 but not add a tag until it is released

Reply:
Beta is fine for now... we are in a weird versioning State, but coming into 5.0 I want to have a solid, tested release that we can build into the future on.